djangocms_maps/templates/djangocms_maps/maps.html

```diff
@@ -1,6 +1,8 @@
 {% load i18n %}{% get_current_language as LANGUAGE_CODE %}
 <div class="djangocms djangocms-maps plugin_maps">
-    {% include "djangocms_maps/maps_title.html" %}
+    {% if object.title %}
+        <h2>{{ object.title }}</h2>
+    {% endif %}
```

Comment:
Please, put everything in a single line, no spacing. The HTML output has unpleasant line breaks and inconsistent indenting otherwise.